ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml

```diff
@@ -93,6 +93,21 @@ tests:
     env:
       PLATFORM_VERSION: "8.4"
     workflow: openshift-e2e-aws-workers-rhel
+- as: e2e-aws-workers-rhel8-nightly
+  interval: 24h
+  steps:
+    cluster_profile: aws
+    env:
+      PLATFORM_VERSION: "8.4"
+    workflow: openshift-e2e-aws-workers-rhel
+- as: unit-nightly
+  commands: |
+    set -o pipefail
+    echo "${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin" >> /etc/passwd
+    tox
+  container:
+    from: test-bin
+  interval: 24h
```

Comment:
Unless each unit run is unique somehow, I think we can leave these as presubmits and drop them from this PR.